### PR TITLE
Added sound option to notification widget

### PR DESCRIPTION
**Pull request by nxnfufunezn**
Fixes : #786 
Need to pass 'audiofile' parameter to  widget.Notify in config file to make this work.

### CODE REVIEW

**Comment by nxnfufunezn** [comment]
review ? @flacjacket

**Comment by flacjacket** [comment]
@nxnfufunezn Sorry I let this one slip through the cracks. I've been
travelling, but I have some time on the plane today I can try to look at
this then.

File: libqtile/widget/notify.py [hunk 1]
```diff
@@ -78,6 +78,8 @@ def set_notif_text(self, notif):
             self.text = '<span weight="bold">%s</span> - %s' % (
                 self.text, pangocffi.markup_escape_text(notif.body)
             )
+        if self.audiofile and path.exists(self.audiofile):
+            self.qtile.cmd_spawn("aplay -q %s" % self.audiofile)
```
Comment (flacjacket) [inline libqtile/widget/notify.py]: Maybe should do `"aplay -q '%s'"` in case the path has spaces in it?

**Comment by flacjacket** [comment]
You should add `audiofile` to the `defaults` list so this option can be documented and given a default value.

**Comment by nxnfufunezn** [comment]
Updated :+1:

**Comment by flacjacket** [comment]
Thanks! I merged this in

**Pull request closed without merging**